benchmark/index.js

```diff
@@ -1,6 +1,9 @@
-const { Suite } = require('benchmark');
-const cursor = require('ansi')(process.stdout);
-const fixtures = require('./fixtures');
+import benchmark from 'benchmark'
+import ansi from 'ansi'
+import fixtures from './fixtures.js'
+
+const { Suite } = benchmark
+const cursor = ansi(process.stdout)
```

Comment:
Wouldn't this work as well?

```suggestion
import { Suite } from 'benchmark'
import ansi from 'ansi'
import fixtures from './fixtures.js'

const cursor = ansi(process.stdout)
```

Reply:
don't remembered if it returned a default object or if the dependency was in cjs that it wasn't able to destruct, but yea it could work